schema/draft/schema.json

```diff
@@ -678,6 +678,9 @@
         },
         "EnumSchema": {
```

Comment:
Mentioned this above in `docs/concepts/elicitation.mdx` repeating here as it needs to be updated as well.

While `enum` is being used properly, `enumNames` is not a valid JSON schema keyword. It was proposed, but never made it into the spec. For standard-compliant ways to provide titles for enumerated values, we can use `oneOf` or `anyOf` with `const` and `title` keywords for each option. 
```JSON
{
  "type": "string",
  "oneOf": [
    {"const": "red", "title": "Red Light"},
    {"const": "yellow", "title": "Yellow Light"},
    {"const": "green", "title": "Green Light"}
  ]
}
```
We are following this recommendation [in the Inspector](https://github.com/modelcontextprotocol/inspector/pull/535#pullrequestreview-2984461045).